fastlane/metadata/it/description.txt

```diff
@@ -0,0 +1,43 @@
+Explore your world, find a quick fact, or dive down a Wikipedia rabbit hole with the official Wikipedia app for iOS. With more than 40 million articles across nearly 300 languages, your favorite free online encyclopedia is at your fingertips. 
```

Comment:
Just double-checking that the `EN` description should be in this `IT` file... same with `DA`...